DirectX12FromScratch/include/MainApplication.h

```diff
@@ -23,4 +23,6 @@ class MainApplication : public MiniEngine::Application
 	MiniEngine::SceneManager	*_sceneManager;
 	MiniEngine::Camera			*_camera;
     MiniEngine::SceneNode       *_node;
+
+	bool _hasFocus;
```

Comment:
Why _hasFocus is here? It's better to have a method in Window for that, don't you think?